src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -120,22 +120,29 @@ const QuickNavigation: React.FC<{
   showAuctionResultsButton: boolean
 }> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
   const { trackEvent } = useTracking()
+
+  const handleQuickNavigationClick = (type: "artworks" | "auction-results") => {
+    const label = type === "artworks" ? "Artworks" : "Auction Results"
+    const destinationPath =
+      type === "artworks" ? "/works-for-sale" : "/auction-results"
+
+    const trackingEvent: SearchSuggestionQuickNavigationItemSelected = {
+      context_module: ContextModule.header,
+      destination_path: href + destinationPath,
+      action: ActionType.searchSuggestionQuickNavigationItemSelected,
+      label,
+    }
+
+    trackEvent(trackingEvent)
+  }
```

Comment:
**nitpick**:  I like your idea of trying to reuse some code. IMHO [this option](https://github.com/artsy/force/pull/12037#discussion_r1126823924) that would be better and more flexible

Reply:
Combined approach

```javascript
const handleArtworksItemClicked = () => {
  trackEvent(tracks.quickNavigationItemTapped({
    destinationPath: "works-for-sale",
    label: "Artworks"
  }))
}

const handleAuctionResultsItemClicked = () => {
  trackEvent({
    destinationPath: "auction-results"
    label: "Auction Results"
  })
}

const tracks = {
  quickNavigationItemTapped: ({ destinationPath, label }: { destinationPath: string, label: string }) => ({
    context_module: ContextModule.header,
    destination_path: `${href}/${destinationPath}`,
    action: ActionType.searchSuggestionQuickNavigationItemSelected,
    label,
  })
}
```